src/helpers/user/user-helper.js

```diff
@@ -4,6 +4,6 @@ const principalApi = getPrincipalApi();
 
 export async function fetchUsers() {
   let usersData = await principalApi.listPrincipals();
```

Comment:
You can return this await emediately
```JS
return await principalApi.listPrincipals();
```

Reply:
This does not even has to be async/await function. the API already returns a promise